app/controllers/accounts_controller.rb

```diff
@@ -9,16 +9,22 @@ class AccountsController < ApplicationController
   before_action :set_user
 
   def edit
-
   end
 
   def update
     changed_notes = []
-    if @user.first_name != account_params[:first_name] || @user.last_name != account_params[:last_name]
-      changed_notes << "Account name was updated."
-    end
-    if @user.email_schedule != account_params[:email_schedule]
-      changed_notes << "Email schedule was updated."
+    changed_notes << 'Account name was updated.' if @user.first_name != account_params[:first_name] || @user.last_name != account_params[:last_name]
+
+    changed_notes << 'Email schedule was updated.' if @user.email_schedule != account_params[:email_schedule]
+
```

Comment:
I think the following should all be behind:
`unless config.never_send_match_summary_email?`
Basically, don't let them change it if the summary emails aren't available.